content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,137 @@
+accomplish through Project Phoenix. Developed in partnership with datavaluepeople, Phoenix is an
+open-source (code repository [here](https://gitlab.com/howtobuildup/phoenix)), end-to-end AI
+platform designed to help peacebuilders and mediators analyze social media data, informing
+effective strategies for conflict prevention and resolution. Since January 2025 phoenix has been
+open to the public and you can apply for a Phoenix account,
+[here](https://howtobuildup.org/phoenix/).
+
+In this article, we’ll explore the challenges Build Up faced, the solutions datavaluepeople
+provided, and how Phoenix is transforming social media analysis for peacebuilding around the globe.
+
+***
+
+## **The challenge: Leveraging social media for peacebuilding**
+
+Peacebuilders and mediators increasingly look to social media to gauge public sentiment and
+identify emerging tensions. Yet collecting and analyzing massive volumes of data from diverse
+platforms can be overwhelming. Several roadblocks stood in the way:
+
+1. **Data overload**: Peacebuilders needed a reliable way to gather, organize, and filter vast
+   streams of social media data in real-time.
+
+2. **Limited technical expertise**: Many peacebuilders aren’t data scientists, so any AI-driven
+   platform had to be intuitive and accessible.
+
+3. **Multi-language support**: Conflicts often span regions with multiple languages, including
+   those that traditional analytics tools rarely handle well.
+
+4. **Open-source requirement**: Build Up wanted a project open to contributors worldwide,
+   facilitating collaboration and continuous improvement.
+
+***
+
+## **The solution: Phoenix—An AI platform for peacebuilders**
+
+Working closely with Build Up, datavaluepeople created Phoenix: a powerful, user-friendly system
+that addresses each challenge in a single, comprehensive platform.
+
+### **Modern web console**
+
+Built with React and Next.js, Phoenix’s web console offers a straightforward interface accessible
+from any device with an internet connection. It allows peacebuilders to easily configure analyses,
+explore data sources, and review insights without advanced technical skills.
+
+### **Authentication and dashboarding**
+
+Phoenix’s custom authentication system ensures secure access, while its Superset-based dashboards
+make data visualization highly flexible. Users can build interactive charts and graphs that help
+surface trends or spikes in conflict-related discussions—providing clarity on community sentiment
+and emerging risks.
+
+### **Data scraping integration**
+
+By integrating with data-scraping platforms like Apify, Phoenix streamlines large-scale data
+collection from social media and other online sources. It lets users define specific parameters for
+what data to capture, helping them focus on the most relevant content while honouring platform and
+data privacy policies.
+
+### **Classification systems for all languages**
```

Comment:
"all languages" might be a bit much...

Reply:
Oh, you are talking about the the text match classification only right? 

Defs don't need the "Unicode supported" bit as that isn't going to mean much to most people. 

I was thinking more like, the other classifiers. But ya if you want to go with "all" :+1: